README.md

```diff
@@ -79,8 +79,38 @@ this.get('store').query('github-release', { repo: 'jimmay5469/old-hash' }); // g
 this.get('store').queryRecord('github-pull', { repo: 'jimmay5469/old-hash', pullId: 1 }); // get a specific pull request
 this.get('store').query('github-pull', { repo: 'jimmay5469/old-hash' }); // get a repo's pull requests
 this.get('store').queryRecord('github-blob', { repo: 'jimmay5469/old-hash', sha: '47c5438403ca875f170db2aa07d1bfa3689406e3' }); // get a file's contents
+this.get('store').queryRecord('github-compare', { repo: 'jimmay5469/old-hash', base: '1234', head: '5678' }); // Compare two commits
 ```
 
+## Testing with Mirage
+
+This addon use [ember-cli-mirage](http://www.ember-cli-mirage.com/) in its tests.  It is often beneficial for consuming apps to be able to re-use the factories and models defined in mirage, so if you would like to use these in your tests you can add the `mirage-support` object to your `ember-cli-build.js` file:
+
+```
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, {
+    ...
+    'mirage-support': {
+      includeAll: true
+    }
+    ...
+  });
+
+  return app.toTree();
+};
+```
+
+
+As long as `ember-cli-mirage` is not disabled, the files in this addon's `mirage-support` directory will be merged with the consuming app's namespace, and be made available to that mirage context.
```

Comment:
Wondering if it's better to link back to mirage docs for this, if it's well-explained there? Otherwise we have to keep this in sync with the source package if there are changes.

Reply:
Unfortunately this is not a first-class thing that is supported by mirage so there is nothing over there to link to. I largely took this implementation from another addon that wanted to do the same thing for its mirage stubs. This is basically manually putting the mirage files into the build tree so consuming apps can access them.

Reply:
Ah ok. Seems fine to leave it here for now. As extra credit, if you feel like it, it might be worth filing a feature request with mirage to support this internally so everyone can benefit from it. (or just submitting a PR to add this along with the issue :P)